### PR TITLE
doc: release: 2.5: Add notes about Atmel, Cypress, UpdateHub, Inventek, TagoIO, Bossac

### PR DESCRIPTION
**Pull request by nandojve**
- Atmel boards and drivers
 - Cypress SoC, boards and drivers
 - UpdateHub improvements
 - Inventek es-WIFI shield
 - TagoIO Network Cloud sample
 - West bossac runner improvements

### CODE REVIEW

**Review by nashif — APPROVED** [review]

File: doc/releases/release-notes-2.5.rst [hunk 1]
```diff
@@ -265,6 +288,13 @@ Build and Infrastructure
     documented in :ref:`dt-from-c`. Information on flash partitions has moved
     to :ref:`flash_map_api`.
 
+* West
+
+  * Improve bossac runner. It supports now native ROM bootloader for Atmel
+    MCUs and extended SAM-BA bootloader like Arduino and Adafruit UF2. The
+    devices supported depends on bossac version inside Zephyr SDK or in users
```
Comment (ioannisg) [inline doc/releases/release-notes-2.5.rst:295]: depend, perhaps?

File: doc/releases/release-notes-2.5.rst [hunk 2]
```diff
@@ -265,6 +288,13 @@ Build and Infrastructure
...
+    path. The recomended Zephyr SDK version is 0.12.0 or newer.
```
Comment (ioannisg) [inline doc/releases/release-notes-2.5.rst:296]: recommended

**Review by ioannisg — CHANGES_REQUESTED** [review]

**Review by MaureenHelm — APPROVED** [review]
Thanks @nandojve

File: doc/releases/release-notes-2.5.rst (hunk 1 above)
Comment (nandojve) [reply doc/releases/release-notes-2.5.rst]: Fixed!

File: doc/releases/release-notes-2.5.rst (hunk 2 above)
Comment (nandojve) [reply doc/releases/release-notes-2.5.rst]: Fixed!

**Comment by nandojve** [comment]
Rebased and fixed both comments from @ioannisg.

**Review by ioannisg — APPROVED** [review]

**Pull request merged**